samples/cmake/enet/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+if(NXDK)
+  # Hack for winsock2.h and mmsystem.h
+  include_directories(SYSTEM "${NXDK_DIR}/lib/winapi/ws2_32;${NXDK_DIR}/lib/winapi/winmm")
```

Comment:
These should be part of default includes; I believe this is an upstream nxdk bug which will have to be integrated in #82